lib/travis/services/find_admin.rb

```diff
@@ -10,11 +10,11 @@ class FindAdmin < Base
       include Travis::Logging
 
       register :find_admin
+      NUM_CANDIDATES = 15
```

Comment:
@cotsog There is no precise reason for choosing 15 alone other than choosing a number large enough for looking up a valid admin, in case quite a few (may be 5 in a row) have be removed.

Since we didn't have any retires so far, we don't know what the best number of retries we'd need.

WDYT?
cc: @svenfuchs